app/controllers/claims_controller.rb

```diff
@@ -10,18 +10,11 @@ class ClaimsController < BasePublicController
   before_action :prepend_view_path_for_policy
 
   def new
-    render first_template_in_sequence
+    persist
```

Comment:
Isn't it an issue that in the `new` call you are creating a claim? I'm assuming we'll have a lot of empty claims just form people who open this page and close it straight away.

Reply:
@vassyz This should clean things up, so shouldn't be an issue.

This means however you rearrange the pages, it should just work then because they all expect a claim and it's present. A little less fragile.